### DISCUSSION TITLE
Debug failed API key test

### DISCUSSION THREAD

**Discussion by ASBaumgarten** [OpenBB-finance/OpenBB#856] [Q&A] [answered]
CoinMarketCap is the only API key that always fails its test. I am using the Podman container version, specifically the `gst-poetry:latest` package. Anyone have suggestions on how to debug this issue?

The script I use to enter the terminal:
```
#!/bin/bash

podman run -it --rm --env-file=/mnt/c/Users/A/Documents/setenv ghcr.io/gamestonkterminal/gst-poetry:latest
```

**Comment by aia** [comment, marked as answer]
I can suggest the following:

1. Launch the container the way you are launching it.
2. Attach to a running container:
```
docker exec -it <container_id> sh
```
3. Create your environment withing the container and launch the terminal through python terminal.py.
4. You can attach VSCode to a running container too to step through with a debugger if you'd like
